src/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
       plugins: ['skyscanner-dates'],
       rules: {
         'skyscanner-dates/no-saddlebag-date': 'warn',
-        'skyscanner-dates/no-moment': 'warn',
+        'skyscanner-dates/no-moment': 'error',
         'skyscanner-dates/no-dayjs': 'warn',
```

Comment:
```suggestion
        'skyscanner-dates/no-dayjs': 'error',
```
Not allow to use `dayjs`.

Reply:
at the beginning, the newly added rule will set to 'warning', and the old rules will keep the same as before, After some months have passed, we will then change the rule to the error severity, to force people to make changes.